apps/src/applab/designMode.js

```diff
@@ -650,6 +656,30 @@ function makeUndraggable(jqueryElements) {
   return foundOne;
 }
 
+/**
+ * Highlights an element with a dashed border, removes border from all other elements
+ */
+function highlightElement(element) {
+  removeElementHighlights();
+
+  if ($(element).is('#designModeViz img,#designModeViz label')) {
+    $(element).parent().css({
+      outlineStyle: 'dashed',
+      outlineWidth: '1px',
+    });
+  }
+}
+
+/**
+ * Remove dashed borders from all elements
+ */
+function removeElementHighlights() {
+  $('#designModeViz div').parent().css({
```

Comment:
How about `$('desigModeViz .ui-draggable).css({...})` ? Although this change looks ok now, I'm concerned about what will happen if we try to put an outline on a div for a different reason down the road.